src/client/Client.cc

```diff
@@ -12553,7 +12553,9 @@ int Client::_statfs(Inode *in, struct statvfs *stbuf,
   lock.lock();
 
   ceph_assert(root);
-  total_files_on_fs = root->rstat.rfiles + root->rstat.rsubdirs;
```

Comment:
Is this suggesting that if the root of the cmount has quota enabled, we will use that root, and not the sub-volume root?

I assume get_quota_root will find the sub-volume root if appropriate.

Reply:
@ffilz Can you confirm this is what you were looking for?